app/views/includes/result_header_subheader.scala.html

```diff
@@ -24,12 +24,12 @@ <h1 class="govuk-heading-l" id="page-title">@Messages("gmp.results.h1")</h1>
 <h2 class="govuk-heading-m" id="results-header">@calculationResponse.header(messages)</h2>
 
 @if(revalRateSubheader.isDefined) {
-    <div id="revaluation-subheader">@Html(revalRateSubheader.get)</div>
+    <div class="govuk-body">@Html(revalRateSubheader.get)</div>
 }
 
 @if(survivorSubheader.isDefined) {
-  <hr class="govuk-section-break govuk-section-break--visible" aria-hidden = true/>
-  <div id="survivor-subheader" class="panel-indent">@survivorSubheader</div>
+  <br>
```

Comment:
Avoid using br tags - if additional spacing is need then there are override classes which can do this.